test_runner/src/main/kotlin/ftl/adapter/google/LocalesAdapter.kt

```diff
@@ -0,0 +1,66 @@
+package ftl.adapter.google
+
+import ftl.api.Locale
+import ftl.environment.TAGS
+import ftl.environment.TestEnvironmentInfo
+import ftl.environment.createTableColumnFor
+import ftl.environment.getOrCreateList
+import ftl.environment.tagToSystemOutColorMapper
+import ftl.run.exception.FlankGeneralError
+import ftl.util.applyColorsUsing
+import ftl.util.buildTable
+
+internal fun List<com.google.testing.model.Locale>.toApiModel(): List<Locale> = map { locale ->
+    Locale(
+        id = locale.id.orEmpty(),
+        name = locale.name.orEmpty(),
+        region = locale.region.orEmpty(),
+        tags = locale.tags.orEmpty()
+    )
+}
+
+fun List<Locale>.asPrintableTable() = createTestEnvironment().createLocalesTable()
```

Comment:
I would move whole logic of printing table to separate file, because we could reuse it later on

Reply:
moved 👍